src/sentry/utils/env.py

```diff
@@ -63,6 +63,26 @@ def log_gcp_credentials_details(logger) -> None:
         },
     )
 
+    try:
```

Comment:
Good idea to move this to the top of the function, just in case we unwittingly add a call somewhere we shouldn't later in debugging, since we don't have any tests to catch this.

Reply:
Can also do this in a follow-up PR, so that we don't have to re-run CI.